buildByond.conf

```diff
@@ -1,7 +1,7 @@
 BYOND_MAJOR_VERSION=514
 BYOND_MINOR_VERSION=1566
 
-SPACEMAN_DMM_VERSION=suite-1.7
+SPACEMAN_DMM_VERSION=suite-1.7.1
```

Comment:
?

Reply:
Addresses lint not knowing about DM particles: https://github.com/goonstation/goonstation/runs/3560370569?check_suite_focus=true

Specifically pulls in: https://github.com/SpaceManiac/SpacemanDMM/pull/275